.mockery.yaml

```diff
@@ -4,7 +4,14 @@ with-expecter: True
 mockname: "{{.InterfaceName}}"
 filename: "{{.MockName}}.go"
 outpkg: mocks
+tags: "custom2"
```

Comment:
What's the purpose of adding this?

Reply:
I check that passing of build tags to package loader works in e2e tests. The IfaceWithCustomBuildTagInComment mock will not be generated without this line.

You can check by commenting this line and exec `task test.e2e`.

Reply:
Ah got it! Makes sense.